docs/zh-cn/manual/device/macos.md

```diff
@@ -9,9 +9,9 @@ icon: basil:apple-solid
 
 ### ✅ [PlayCover](https://playcover.io)（原生运行最流畅 🚀）
 
-试验性支持，遇到问题请多多提 issue，并在标题中提及 iOS。
+试验性支持，遇到问题请多多提 issue，并在标题中提及 `Mac` 。
```

Comment:
https://github.com/MaaAssistantArknights/issue-checker/blob/08d909e163099974c55f9dda40e45ac668bf91dc/.github/issue-checker.yml#L303-L304

This definitely clears up the idea of using MAA on iOS (which is the mobile OS) compared to MacOS which is the computer one, where MAA actually runs, but just as a heads up, the issue-checker seems a bit picky, especially on what capitalization is used. For example, this is not correct :P
![image](https://github.com/user-attachments/assets/e965d7c1-01fc-4b2c-9c75-c70c7869ae30)

Reply:
then fix the issue checker. I forgot to update the en version for this doc.

Reply:
Whenver you have time :)

Reply:
our issue-checker's regex engine cannot directly specify to ignore case
so can only make it like `[Mm][Aa][Cc][Oo][Ss]`
holy sh*t